caravel/models.py

```diff
@@ -1265,6 +1243,59 @@ def recursive_get_fields(_conf):
             query=query_str,
             duration=datetime.now() - qry_start_dttm)
 
+    @staticmethod
+    def get_filters(filter):
```

Comment:
`filter` is a standard lib, main namespace function. Here you are redefining it for the scope of this function. Use `filter_` or some other name instead.